examples/mongodbatlas_stream_privatelink_endpoint/confluent_serverless/main.tf

```diff
@@ -0,0 +1,59 @@
+resource "confluent_environment" "staging" {
+  display_name = "Staging"
+}
+
+resource "confluent_private_link_attachment" "pla" {
+  cloud        = "AWS"
+  region       = var.aws_region
+  display_name = "private-link-attachment"
+  environment {
+    id = confluent_environment.staging.id
+  }
+}
+
+module "privatelink" {
```

Comment:
Curious why we are using a module here

Reply:
No reason apart from the fact that it makes the `main.tf` smaller

Reply:
I would move it to a top level file `privatelink.tf` instead of the module then.